sdk/confidentialledger/azure-security-confidentialledger/README.md

```diff
@@ -3,7 +3,7 @@
 Azure Confidential Ledger provides a service for logging to an immutable, tamper-proof ledger. As part of the [Azure Confidential Computing][azure_confidential_computing]
 portfolio, Azure Confidential Ledger runs in SGX enclaves. It is built on Microsoft Research's [Confidential Consortium Framework][ccf].
 
-**Please rely heavily on the javadocs and our [Low-Level client docs][low_level_client] to use this library**
+**Please rely heavily on the javadocs to use this library**
```

Comment:
Let's just remove this line for now, unless we have links to some specific javadocs we can point to.

Reply:
Agreed

Reply:
Okay, this line has been removed for now.